### DISCUSSION TITLE
Choice between graphQL and REST fastAPI

### DISCUSSION THREAD

**Discussion by Matthieu-Tinycoaching** [fastapi/fastapi#6604] [Questions] [answered]
Hi community,

I have a quick question that turns me on for a while. I work with both graphQL and fastAPI, however I can see through documentation and tutorials that REST API are by far more usual. Are they definitely better than graphQL? Since I just started my graphQL API I wonder whether I should move on REST or keep going on with strawberry?
Thanks for your lights!

**Comment by mustadev** [comment, marked as answer]
you should use Discussion. this is not a issue

**Comment by Matthieu-Tinycoaching** [comment]
Thanks I have posted this in discussion.
